todo_app/app.py

```diff
@@ -8,4 +9,13 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template("index.html", items=items)
+
+
+@app.route('/add_items', methods=["POST"])
```

Comment:
You have used a plural here, but it only creates 1 item at a time.

Reply:
Noted, thank you will change!